contracts/PortfolioLib.sol

```diff
@@ -65,6 +65,7 @@ error NegativeBalance(address token, int256 net);
 error NotController();
 error NonExistentPool(uint64 poolId);
 error NonExistentPosition(address owner, uint64 poolId);
+error NotPerpetualPool();
```

Comment:
nit: this error name is somewhat confusing...it would make me think the revert happened because this pool is not a perpetual pool, when in fact the revert happens because it _is_ a perpetual pool

Reply:
Fixed